lib/commands/serve.js

```diff
@@ -56,15 +57,16 @@ module.exports = function (
         .watch(path)
 
       if (openInBrowser) {
-        open(`http://localhost:${port}`)
+        const host = ip === '0.0.0.0' ? 'localhost' : ip
```

Comment:
it would break if the user wants it to listen to `0.0.0.0`

I guess it would be fine to simple add `host = ip` ?

Reply:
This `host` only determines which URL the browser will open to. Browsers cannot connect to `0.0.0.0` which is why we need this override. It will not break things since if the server is listening on `0.0.0.0`, it will also work if the browser uses `localhost`.

Reply:
thanks for this detail.